[CC][master] Improvements to CC docs (#5178)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/tls/component-certificates.md

    ```

    ``` tab="enforcer"
    <CHOREO-CONNECT_HOME>/docker-compose/resources/enforcer/security/truststore
    ```

    ``` tab="router"
    <CHOREO-CONNECT_HOME>/docker-compose/resources/router/security/truststore
    ```
        
3.  Restart the component.
   
    `docker restart <container_name>`
   

!!! info

    The certificate locations are configured as a volume mount for each component container in the `docker-compose.yaml` file as given below.

    ``` tab="adapter"
    volumes:
            - ../resources/adapter/security:/home/wso2/security
    ```

    ``` tab="enforcer"
    volumes:
            - ../resources/enforcer/security:/home/wso2/security
    ```

    ``` tab="router"
    volumes:
            - ../resources/router/security:/home/wso2/security
    ```

    This mounts both the key store and truststore locations to the container.

## Changing the Private Certificate of a Component
If it is required to change the private certificate of a component, follow the steps below.

1. Generate a new key pair for the component.
   
2. Copy the private key and certificate (in PEM format) into `<CHOREO-CONNECT_HOME>/resources/<COMPONENT>/security/keystore` directory.

>     Ex: To change the private key of Adapter component, copy the new certificate to `<CHOREO-CONNECT_HOME>/resources/adapter/security/keystore` directory.

Reviewer: ```suggestion
    Example: 
    
    To change the private key of the Adapter component, copy the new certificate to  the `<CHOREO-CONNECT_HOME>/resources/adapter/security/keystore` directory.
```

> 3. If the private key file name is changed, edit the corresponding configuration in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` or `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect-with-apim/conf/config.toml` file. Otherwise, use the same name as `mg.pem` and `mg.key` for the new certificate and key.

Reviewer: ```suggestion
3. If you change the private key file name, edit the corresponding configuration in the `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` or `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect-with-apim/conf/config.toml` file. Otherwise, use the same name as `mg.pem` and `mg.key` for the new certificate and key.
```

4. Copy the public certificate (in PEM format) into the other two components. `<CHOREO-CONNECT_HOME>/resources/<COMPONENT>/security/truststore`

>     Ex: Copy the certificate to `<CHOREO-CONNECT_HOME>/resources/enforcer/security/keystore` and `<CHOREO-CONNECT_HOME>/resources/router/security/keystore`

Reviewer: ```suggestion
    Example: 
    
    Copy the certificate to `<CHOREO-CONNECT_HOME>/resources/enforcer/security/keystore` and `<CHOREO-CONNECT_HOME>/resources/router/security/keystore`
```
